StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/CVCReconfirmationView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  CVCReconfirmationView.swift
+//  StripePaymentSheet
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+@_spi(STP) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+class CVCReconfirmationView: UIView {
```

Comment:
supernit you don't need to fix, but did you mean to call this `CVCRecollectionView` since the element is `CVCRecollectionElement`?